### ISSUE TITLE
Bug in splitting the options line

### ISSUE THREAD

**Issue by haozhun** [tmikov/hermes-explorer-page#1]
If I remove all options, I get "Multiple files must use CommonJS modules."

**Comment by tmikov** [comment]
Interesting. Why isn't https://github.com/tmikov/hermes-explorer-page/blob/456f0200b44280f01e555f88cd5e71c8af13def0/explorer.js#L59 working...

**Comment by haozhun** [comment]
```
d8> ''.split(/\s+/)
[""]
d8> '1 2'.split(/\s+/)
["1", "2"]
d8> '1'.split(/\s+/)
["1"]
d8> ' 1 '.split(/\s+/)
["", "1", ""]
```

Note case 1 and 4.

There is no obvious winner here in terms of which is the least surprising behavior of `split` for case 1.

**Comment by tmikov** [comment]
Fixed in e70d14ecbdf4f6c7df3e60699e39c93875d06eca by adding `.filter(x => x)`.
